controllers/gce/backends/backends.go

```diff
@@ -26,10 +26,10 @@ import (
 	"github.com/golang/glog"
 
 	compute "google.golang.org/api/compute/v1"
+	api_v1 "k8s.io/api/core/v1"
```

Comment:
Is there a reason to merge these two blocks? I really like the separation between the k8s API imports and other imports